### ISSUE TITLE
onyxsdk-base could not determine dependecies

### ISSUE THREAD

**Issue by zbouslikhin** [onyx-intl/OnyxAndroidDemo#86]
Hello,

I'm trying to use onyxsdk-pen, which uses onyxsdk-base

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:compileDebugJavaWithJavac'.
> Could not resolve all task dependencies for configuration ':app:debugCompileClasspath'.
   > Could not find pub.devrel:easypermissions:0.2.1.
     Required by:
         project :app > com.onyx.android.sdk:onyxsdk-base:1.6.19
   > Could not find com.tencent:mmkv:1.0.19.
     Required by:
         project :app > com.onyx.android.sdk:onyxsdk-base:1.6.19
```

**Comment by Wiskerke** [comment]
I had the same problem.

If you look at build.gradle file in the archive, it contains a dependency on jcenter: 

https://github.com/onyx-intl/OnyxAndroidDemo/blob/005b4686dc285f33fecbecfd5c9863b2e8b462bf/build.gradle#L3-L8

Now in android studio, it gives a hint to replace this: 
![image](https://user-images.githubusercontent.com/1465986/180857786-50cff2ed-a254-48de-99be-143fa22885ee.png)

However, mavenCentral does not contain those two dependencies. (It contains newer versions, but not those old versions.)
So basically, make sure that `jcenter()` is present in the gradle file.
